defaultTheme.js

```diff
@@ -169,7 +167,9 @@ module.exports = function() {
       '4': '4px',
       '8': '8px',
     },
-    borderColors: global.Object.assign({ default: colors['grey-light'] }, colors),
+    borderColors: theme => {
+      return global.Object.assign({ default: theme.colors['grey-light'] }, theme.colors)
```

Comment:
Just curious why you use `global`. Are you worried about `Object` existing in the local namespace? Seems pretty unlikely...

Reply:
Very reasonable question! Check this out: https://github.com/tailwindcss/tailwindcss/pull/402